packages/preview-button/admin/src/utils/index.js

```diff
@@ -1 +1,4 @@
+export { default as getTrad } from './getTrad';
 export { default as pluginId } from './plugin-id';
+export { getContentByType } from './getContentByType';
+export { getUrl } from './getUrl';
```

Comment:
Ondersteunen deze packages geen TypeScript?

Reply:
Helaas niet. Wat denk je ervan als ik deze package in een andere PR naar TypeScript converteer?